<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="0" colspan="2">
     <widget class="QTextEdit" name="DisplayResult">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="fun2_2">
      <property name="text">
       <string>New Youk NY</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="QPushButton" name="loadData">
      <property name="text">
       <string>load data</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Year:</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Origin City:</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="lineEdit_11">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Destination City:</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLineEdit" name="fun3_1"/>
    </item>
    <item row="8" column="2" colspan="4">
     <widget class="QFrame" name="chart">
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="fun1_1">
      <property name="text">
       <string>Atlanta GA</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="fun2_1">
      <property name="text">
       <string>Chicago IL</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLineEdit" name="lineEdit_13">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>City Name:</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="fun1_2">
      <property name="text">
       <string>Los Angeles CA</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="fun1_3">
      <property name="text">
       <string>2001</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Origin City:</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="6">
     <widget class="QLineEdit" name="Display">
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string>Please load data first!</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Destination City:</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_1">
      <property name="text">
       <string>function 1</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2" colspan="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>function 2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="reset">
      <property name="text">
       <string>reset graphic</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4" colspan="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>function 3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>Display</tabstop>
  <tabstop>loadData</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>fun1_1</tabstop>
  <tabstop>fun1_2</tabstop>
  <tabstop>fun1_3</tabstop>
  <tabstop>lineEdit_9</tabstop>
  <tabstop>lineEdit_11</tabstop>
  <tabstop>fun2_1</tabstop>
  <tabstop>fun2_2</tabstop>
  <tabstop>lineEdit_13</tabstop>
  <tabstop>fun3_1</tabstop>
  <tabstop>pushButton_1</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
